lib/shopify_dashboard_plus.rb

```diff
@@ -142,137 +178,173 @@ def hash_to_graph_format(sales)
       end
     end
 
+    # This hash will return repeated values (i.e., :data => [["item 1", 6], ["item 1", 6]])
+    # ChartKick will ignore repeated entries, so the totals need to be merged
+    # i.e., :data => [["item1", 12]]
+    if merge_results
+      name_hash.each_with_index do |item, index|
+        consolidated_data = Hash.new(0)
+        item[:data].each do |purchase_entry|
+          consolidated_data[purchase_entry[0]] = consolidated_data[purchase_entry[0]].plus(purchase_entry[1])
+        end
+        name_hash[index][:data] = hash_to_list(consolidated_data)
+      end
+    end
+
     name_hash
   end
 
+  # Return order query parameters hash
+  def order_parameters_paginate(start_date, end_date, page)
+    desired_fields = ["total_price", "created_at", "billing_address", "currency", "line_items", "customer", "referring_site", "discount_codes"]
```

Comment:
I'd put this in a constant somewhere

```ruby
DESIRED_FIELDS = [
  # ...
]
```

seen as it doesn't change in the method.